spatial-tracking-explainer.md

```diff
@@ -171,27 +170,13 @@ function onSessionStarted(session) {
 }
 ```
 
-### Identity reference spaces
-An _identity_ reference space is one which provides no tracking. This type of reference space is used for creating inline experiences with tracking information explicitly disabled. Instead, developers use `XRReferenceSpace.originOffset` which is described in the [Application supplied transforms section](#application-supplied-transforms).  An example usage of an _identity_ reference space is a furniture viewer that will use [click-and-drag controls](#click-and-drag-view-controls) to rotate the furniture. It also supports cases where the developer wishes to avoid displaying any type of tracking consent prompt to the user prior while displaying inline content.
-
-This type of reference space is requested with a type of `identity` and returns a basic `XRReferenceSpace`. `XRViewerPose` objects retrieved with this reference space will have a `transform` that is equal to the reference space's `originOffset` and the `XRView` matrices will be offset accordingly.
+Sometimes it is useful to have access to the `XRSpace` represented by the viewer directly, such when the developer wants to use it to compare locations against other `XRSpace` objects.
 
 ```js
-let xrSession = null;
-let xrReferenceSpace = null;
-
-// Create an 'identity' reference space
-function onSessionStarted(session) {
-  xrSession = session;
-  xrSession.requestReferenceSpace('identity')
-  .then((referenceSpace) => {
-    xrReferenceSpace = referenceSpace;
-  })
-  .then(setupWebGLLayer)
-  .then(() => {
-    xrSession.requestAnimationFrame(onDrawFrame);
-  });
-}
+  let pose = xrFrame.getPose(preferredInputSource.gripSpace, xrSession.viewerSpace);
```

Comment:
xrSession.viewerSpace doesn't exist with this PR anymore, right?

Reply:
Correct, thanks.